pom.xml

```diff
@@ -0,0 +1,51 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.nuxeo</groupId>
+    <artifactId>nuxeo-addons-parent</artifactId>
+    <version>9.10-SNAPSHOT</version>
+  </parent>
+
+  <groupId>org.nuxeo.jsf.to.web.ui</groupId>
```

Comment:
This group seems a bit fancy. Please use a simple and existing one like `org.nuxeo.web.ui` instead (also update the other commit in addons)